build.gradle

```diff
@@ -96,10 +96,8 @@ dependencies {
     compile 'com.opencsv:opencsv:3.4'
     compile 'com.google.guava:guava:18.0'
     compile 'com.github.samtools:htsjdk:'+ htsjdkVersion
-    // Using the shaded version to avoid conflicts between its protobuf dependency
-    // and that of Hadoop/Spark (either the one we reference explicitly, or the one
-    // provided by dataproc).
-    compile 'com.google.cloud:gcloud-java-nio:0.2.8:shaded'
+    // Don't use the shaded version, since it includes un-relocated netty, which causes conflicts with Spark
+    compile 'com.google.cloud:gcloud-java-nio:0.2.8'
```

Comment:
Did you try this with Dataproc?

I'm getting a strange error:
ERROR: (gcloud.dataproc.jobs.submit.spark) You do not have permission to access cluster [jp-spark-cluster].

This may be related to recent changes to gcloud:
WARNING: `gcloud auth login` no longer writes application default credentials.